### DISCUSSION TITLE
Dart SDK for Flutter

### DISCUSSION THREAD

**Discussion by joshbrgs** [open-feature/community#378] [Ideas]
I see OpenFeature is starting to create client-side SDKs and see that the native mobile SDKs are available and the React SDK may be possible for React Native, so I wanted to throw out the idea of a Flutter package written in Dart for the support of this framework as well. It could be an interesting contribution and love where OpenFeature is going!

**Comment by nicklasl** [comment]
Personally I don't have a clear idea on how widespread Flutter is and how its future looks but I guess one could get started pretty fast if it would use the iOS and Android SDKs under the hood?

**Reply by joshbrgs** [reply]
I am also unsure what the future looks like for Flutter, but have seen some acceptance from the industry as far as how it has been trending (it has just surpassed React Native in GitHub stars). I think it would have to be a new Dart project, but following the implementation of how the Kotlin or Swift SDK implements logic.

**Comment by lukas-reining** [comment]
Hey @joshbrgs, we have talked about having Flutter several times in the community meetings and other talks around.
We had many people supporting the idea, but no one started on it yet.
So yes I think it would be great to have in the project and support it, we currently have no one who started on it.

**Reply by joshbrgs** [reply]
I'd love to contribute and at least start on the Flutter package. Is there a repo I can push the initial commits to, and what would you want pushed as far as initial commits?

**Reply by beeme1mr** [reply]
Someone started a Flutter SDK a while ago but it looks like the conversation trailed off. Here's [the thread](https://github.com/open-feature/spec/issues/180) where we talked about it. Perhaps it's worth reviewing what has been done so far to see what's remaining. We could add another repo to the org if someone is willing to help maintain it.
